build/.vscode/launch.json

```diff
@@ -1,28 +0,0 @@
-{
```

Comment:
I'm not sure why these files have been deleted, but it'd be great to keep them around.